httpbench.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime
+
+from scrapy import Request, Spider
+
+
+class HTTPSpider(Spider):
+    """Spider equivalent to https://http1.golang.org/gophertiles
+
+    Use the DOWNLOAD_HANDLERS setting to set the download handler to test.
+    """
+    name = 'httpbench'
+
+    def start_requests(self):
+        self.response_count = 0
+        self.start_time = datetime.utcnow()
+        version = (
+            '2' if '2' in self.settings.getwithbase('DOWNLOAD_HANDLERS')['https']
+            else '1'
+        )
+        for x in range(14):
+            for y in range(11):
+                yield Request(
+                    f'https://http{version}.golang.org/gophertiles?x={x}&y={y}&latency=0'
```

Comment:
Here and below f-strings are used which is python 3.6 syntax, while in ``tox.ini`` we test under py27 -- py35, so it would be best to either update tox.ini to drop python support for python < 3.6 (I think it's completely fine now), or use python2-compatible syntax.